charts/prefect-orion/templates/service.yaml

```diff
@@ -26,7 +26,7 @@ spec:
   externalTrafficPolicy: {{ .Values.service.externalTrafficPolicy | quote }}
   {{- end }}
   ports:
-    - name: orion # will this be fine?
+    - name: orion-svc-port
```

Comment:
there's also an appProtocol field, which is useful to set to 'http' or 'https' depending what we have here https://kubernetes.io/docs/concepts/services-networking/service/#application-protocol